### ISSUE TITLE
Negative scale does not play nicely with GE

### ISSUE THREAD

**Issue by StjerneIdioten** [StjerneIdioten/I3D-Blender-Addon#6]
Objects that are mirrored by having a negative scale does not export properly to GE.
![mirror_flipped_normals](https://user-images.githubusercontent.com/11218511/73601740-14bb7900-4567-11ea-81ae-530f586b2532.png)

**Comment by StjerneIdioten** [comment]
@NMC-TBone Any thoughts on this? It is only a problem for collection instances as for everything else we can just post a warning saying to not use negative scale for objects, but instances can only be mirrored like that as far as I am aware (unless something has changed)

**Comment by NMC-TBone** [comment]
> [@NMC-TBone](https://github.com/NMC-TBone) Any thoughts on this? It is only a problem for collection instances as for everything else we can just post a warning saying to not use negative scale for objects, but instances can only be mirrored like that as far as I am aware (unless something has changed)

I actually had this open when you sent the comment 😂Negative scale is something I try to avoid at all costs. Yea exactly, thats the only way to mirror a instance.

I have never heard about anyone that really want this to work, so not sure if the time required to solve it is worth it in the end 😅

**Comment by StjerneIdioten** [comment]
> I have never heard about anyone that really want this to work, so not sure if the time required to solve it is worth it in the end 😅

True, but as far as I understand spawning collection instances isn't necessarily out there when it comes to working in Blender now and I guess you probably want them mirrored sometimes. But it can definitely be achieved as well in other ways in Blender that don't lead to this annoying issue.

**Comment by StjerneIdioten** [comment]
This seems to not be reproduce-able in newer versions of the export as matrix calculations for the exported rotations seems to correct the negative scaling
